Kernel/KSyms.cpp

```diff
@@ -173,7 +173,7 @@ NEVER_INLINE static void dump_backtrace_impl(FlatPtr base_pointer, bool use_ksym
 
 void dump_backtrace_from_base_pointer(FlatPtr base_pointer)
 {
-    dump_backtrace_impl(base_pointer, g_kernel_symbols_available, PrintToScreen::Yes);
+    dump_backtrace_impl(base_pointer, false, PrintToScreen::No);
```

Comment:
The `false` looks very mysterious at the call site. Do `..., /*use_ksym=*/false, …`, that makes it clearer.

(It'd be even better if the param was an enum, not a bool, but that's existing code.)

Reply:
Yep, this is existing code using bools which I also don't like, but 🤷

Reply:
resolved but ignored comment

Reply:
I've never actually seen this inline style of comment in the Serenity source code, but I'll use it. I also added a FIXME to change the signature of `dump_backtrace_impl` to use an enum instead of a bool.

Reply:
There's a couple in LibJS:

https://github.com/SerenityOS/serenity/blob/0f3e1668bb1606bd6c9af6f028035ebafc014cb0/Userland/Libraries/LibJS/Parser.cpp#L873
https://github.com/SerenityOS/serenity/blob/0f3e1668bb1606bd6c9af6f028035ebafc014cb0/Userland/Libraries/LibJS/Parser.cpp#L1360

...but ideally we'd write all code in a way that they are never needed :)